src/main/java/seedu/address/model/person/StudentId.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class StudentId {
+
+    public static final String MESSAGE_CONSTRAINTS = "Student Numbers should start with an A, "
+            + "followed by 7 numbers, then end with an alphabet.";
+
+    /*
+     * The first character of the Student ID must be an A,
+     * followed by 7 digits, then an alphabet.
```

Comment:
Maybe we should emphasize that it must end with a capital alphabet.

Reply:
Okay!